test-files/cmd/slow/NetworkConflateCmdRelationEdgeTest/out.osm

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <osm version="0.6" generator="hootenanny" srs="+epsg:4326">
```

Comment:
This test output added a few reviews as a result of Generic Line Conflation now occurring. I believe one of them isn't valid and should be dealt with in #3718.